client/reader/controller.js

```diff
@@ -296,7 +293,10 @@ module.exports = {
 					onClose: function() {
 						page.back( context.lastRoute || '/' );
 					},
-					onClosed: removeFullPostDialog
+					onClosed: removeFullPostDialog,
+					onPostNotFound: function() {
+						module.exports.sidebar( context, next )
```

Comment:
I don't like this very much, but how else would I display sidebar from within `reader/full-post` component?